### PR TITLE
Could not find remote branch 0.12.0 to clone

### PR DESCRIPTION
**Pull request by ammer**
[!] Error installing react-native-camera
[!] /usr/local/bin/git clone https://github.com/lwansbrough/react-native-camera /var/folders/dr/7kdz8qn91b19mn2h1zcfs60c0000gn/T/d20171201-58908-1njx0ag --template= --single-branch --depth 1 --branch 0.12.0

Cloning into '/var/folders/dr/7kdz8qn91b19mn2h1zcfs60c0000gn/T/d20171201-58908-1njx0ag'...
warning: Could not find remote branch 0.12.0 to clone.
fatal: Remote branch 0.12.0 not found in upstream origin

### CODE REVIEW

**Comment by ospfranco** [comment]
Anybody with more experience with cocoapods could review this

File: react-native-camera.podspec [hunk 1]
```diff
@@ -10,7 +10,7 @@ Pod::Spec.new do |s|
   s.license        = package['license']
   s.author         = package['author']
   s.homepage       = package['homepage']
-  s.source         = { :git => 'https://github.com/lwansbrough/react-native-camera', :tag => s.version }
+  s.source         = { :git => 'https://github.com/lwansbrough/react-native-camera', :tag => "v#{s.version}" }
```
Comment (sibelius) [inline react-native-camera.podspec]: this should change: https://github.com/react-native-community/react-native-camera

Comment (sibelius) [reply react-native-camera.podspec]: @ammer do you wanna update this?

Comment (bkvirendra) [reply react-native-camera.podspec]: updated this in PR https://github.com/react-native-community/react-native-camera/pull/1256

**Comment by ptomasroos** [comment]
I can help out with cocoapods. Done a fair share of podspecs for rn

**Comment by sibelius** [comment]
Fixed in another pr

**Pull request closed without merging**